scalecube-configuration/src/main/java/io/scalecube/configuration/authorization/Role.java

```diff
@@ -1,4 +1,4 @@
-package io.scalecube.configuration;
+package io.scalecube.configuration.authorization;
 
 public enum Role {

```

Comment:
Again, not clear why we need this string here.. in worse casu one can use native`name()` methon if String is needed... 
And, let's keep it capital

Reply:
Role class is removed